columnflow/tasks/framework/mixins.py

```diff
@@ -781,52 +781,6 @@ class PlotMixin(AnalysisTask):
         description="a command to execute after the task has run to visualize plots right in the "
         "terminal; no default",
     )
-    skip_ratio = luigi.BoolParameter(
-        default=False,
-        significant=False,
-        description="when True, no ratio (usually Data/Bkg ratio) is drawn in the lower panel; "
-        "default: False",
-    )
-    shape_norm = luigi.BoolParameter(
-        default=False,
-        significant=False,
-        description="when True, each process is normalized on it's integral in the upper panel; "
-        "default: False",
-    )
-    skip_legend = luigi.BoolParameter(
-        default=False,
-        significant=False,
-        description="when True, no legend is drawn; default: False",
-    )
-    skip_cms = luigi.BoolParameter(
-        default=False,
-        significant=False,
-        description="when True, no CMS logo is drawn; default: False",
-    )
-    y_scale = luigi.ChoiceParameter(
-        choices=(law.NO_STR, "linear", "log"),
-        default=law.NO_STR,
-        significant=False,
-        description="string parameter to define the y-axis scale of the plot in the upper panel; "
-        "choices: NO_STR,linear,log; no default",
-    )
-    process_lines = law.CSVParameter(
-        default=(),
-        description="comma-separated process names or patterns to plot as individual lines; "
-        "can also be the key of a mapping defined in the 'process_groups' auxiliary data of "
-        "the config; uses no process when empty; empty default",
-    )
-
-    def get_plot_parameters(self):
-        # convert parameters to usable values during plotting
-        return {
-            "skip_ratio": self.skip_ratio,
-            "shape_norm": self.shape_norm,
-            "skip_legend": self.skip_legend,
-            "skip_cms": self.skip_cms,
-            "yscale": None if self.y_scale == law.NO_STR else self.y_scale,
-            "process_lines": self.process_lines,
-        }
```

Comment:
Maybe we define a base method here to express that this feature exists? Also, in case we have generic parameters that can be defined in this main mixin, we would be able to inject those here. I.e.

```python
def get_plot_parameters(self):
    return {}
```

And then in `PlotBase` (inheriting from this mixin) and below, we start with `parameters = super().get_plot_parameters()` and update `parameters` (edit: like you already do actually :) but we could start at this mixin)

Reply:
Thinking about this again ... do we still need the mixin when all plots anyway inherit from the new `PlotBase`, i.e., would there at some point be any task that inherits from the mixin but not the base tasks?

Reply:
You're probably right. And even if the `PlotBase` is not the true base task for all plots, we simply have to modify it or add another PlotBase task somewhere.